src/Geoscape/DogfightState.cpp

```diff
@@ -1641,7 +1616,7 @@ void DogfightState::update()
 				}
 				else
 				{
-					_ufo->setSecondsRemaining(RNG::generate(24, 96)*3600);
+					_ufo->setSecondsRemaining(static_cast<size_t>(RNG::generate(24, 96)) * 3600);
```

Comment:
something that could be data driven, but isn't(yes, I know, you are just copying existing functionality)... sure, there's always going to be 24 hours in a day, but perhaps someone might want an event to last 100 days. Something to think about.

Reply:
Definitely agree.